### PR TITLE
Completed prep-html-links | Chris Mendoza

### PR DESCRIPTION
**Pull request by CTMendoza**

### CODE REVIEW

File: prep-html-links/contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+<a href="file:///C:/Users/Topper/Downloads/prep-html-links/index.html"> Click here to go to the home page</a>
```
Comment (thebearingedge) [inline prep-html-links/contact-us/contact-us.html]: This won't work on somebody else's computer. You'll want to use a relative path back to the index file.

```html
<a href="../index.html"> ... </a>
```

File: prep-html-links/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="file:///C:/Users/Topper/Downloads/prep-html-links/contact-us/contact-us.html"> Click here to go to the contact page </a>
```
Comment (thebearingedge) [inline prep-html-links/index.html]: Here you can use a relative path to the `contact-us.html` file inside of the `contact-us` directory.

```html
<a href="contact-us/contact-us.html"> ... </a>
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-html-links/index.html (hunk 2 above)
Comment (CTMendoza) [reply prep-html-links/index.html]: Updated. Thanks for the feedback Tim. I think I grasp it better. The examples using img src kind of threw me off.

Prep-Html_links2

File: prep-html-links/contact-us/contact-us.html (hunk 1 above)
Comment (CTMendoza) [reply prep-html-links/contact-us/contact-us.html]: will <a href="prep-html-links/index.html"> ... </a> work here as well?

Comment (CTMendoza) [reply prep-html-links/contact-us/contact-us.html]: will `<a href="prep-html-links/index.html"> ... </a>` work here as well?

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**